backend/battle/tests.py

```diff
@@ -1,3 +1,46 @@
-# from django.test import TestCase
+from django.test import TestCase, Client
+from model_bakery import baker
+from users.models import User
+from battle.models import Battle
 
-# Create your tests here.
+'''
+class ListBattlesTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.battle = baker.make('battle.Battle')
+        self.user = User.objects.create(email='deborah.mendonca@vinta.com.br', password='admin')
+        self.user.set_password('admin')
+        self.user.save()
+    
+    def test_login_user_can_acess_battle_list(self):
+        # import ipdb; ipdb.set_trace()
+        self.client.login(username=self.user.email, password='admin')
+        response = self.client.get('/battle/list/')
```

Comment:
You can and should use Django's `reverse('namespace')` to make the tests future proof

Reply:
got it ;)